crates/paimon/src/spec/types.rs

```diff
@@ -98,12 +98,217 @@ pub enum DataType {
     Row(RowType),
 }
 
+impl DataType {
+    fn is_nullable(&self) -> bool {
+        match self {
+            DataType::Boolean(_type) => _type.nullable,
```

Comment:
The `_` prefix in `_type` is often seen as denoting unused values. To avoid confusion, we can simply use `v` instead of `_type`.